Update the Managing a Shopping Cart page and the Creating an Order page from the Shopping Assistance block (#1303)
magento/merchdocs: src/customers/customer-account-create-order.md

> ### Add products from the Customer’s Activities section

Reviewer: Let's change this while we are in here

```suggestion
### Add products from the Customer’s Activities sidebar
```

> Items can be transferred to the cart from a customer wish list, as well as recently viewed, compared, or ordered items.

Reviewer: ```suggestion
You can transfer items to the cart from a customer's wish list, as well as recently viewed, compared, or ordered items.
```

1. Expand ![]({% link images/images/btn-expand.png %}) one of the following sections.

   - Wish List
   - Last Ordered Items
   - Products in Comparison List
   - Recently Compared Products
   - Recently Viewed Products

1. Select the checkbox of each product in the left panel.

> 1. Scroll down and click **Update Changes**.

Reviewer: Looks like this has changed

```suggestion
1. Scroll down and click <span class="btn">Update Changes</span>.
```

   - Click <span class="btn">Add Selected Product(s) to Order</span> to update the cart.

> 1. Configure the product [gift options]({% link sales/gift-options.md %}):
> 
>    - Click **Gift Options**.
> 
>    - Complete the `Gift Message` fields or select the `Gift Wrapping`.
> 
>    - Click <span class="btn">OK</span>.

Reviewer: I think it is simpler to just link to this since it depends on the product configuration.

```suggestion
1. If a product is configured for [gift options]({% link sales/gift-options.md %}), set the options as needed.
```
